libs/filabridge/include/filament/MaterialChunkType.h

```diff
@@ -101,6 +101,8 @@ enum UTILS_PUBLIC ChunkType : uint64_t {
     DictionaryMetalLibrary = charTo64bitNum("DIC_MLIB"),
 
     MaterialCrc32 = charTo64bitNum("MAT_CRC "),
+
+    MaterialSource = charTo64bitNum("MAT_SRC"),
```

Comment:
I think these chunk types need to be 8 chars long (see `MAT_CRC `) above. So we need a space at the end.

```suggestion
    MaterialSource = charTo64bitNum("MAT_SRC "),
```